lib/analyzer/apk-analyzer.ts

```diff
@@ -20,10 +20,10 @@ function getPackages(targetImage: string) {
     .then(parseFile);
 }
 
-function parseFile(text: string) {
+function parseFile(output: Output) {
```

Comment:
don't think this function should get `Output` - just give it the text like before.